src/card/card.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let id = '';
+  export let imgSrc = '';
```

Comment:
The only point in taking images into consideration is that everything inside a card is padded, but images wouldn't look good. The design we are using for the product card in Innopoints wouldn't be possible otherwise

Reply:
I also think it is not uncommon that people would add at least a title to the card. It does no harm, just one line for convenience

Reply:
A title to the card? Never seen anything like it. It does bring clutter and possibly harm Svelte optimizations. I'd say we remove it, for I really see no purpose for it.